src/NServiceBus.Hosting.Profiles/IConfigureLoggingForProfile.cs

```diff
@@ -1,15 +1,12 @@
-﻿namespace NServiceBus.Hosting.Profiles
+﻿// ReSharper disable PartialTypeWithSinglePart
+namespace NServiceBus.Hosting.Profiles
 {
     using NServiceBus;
 
     /// <summary>
     /// Configure logging in the constructor of the class that implements IConfigureThisEndpoint.
     /// </summary>
-    [ObsoleteEx(
-        TreatAsErrorFromVersion = "7.0",
-        Message = "Configure logging in the constructor of the class that implements IConfigureThisEndpoint.",
-        RemoveInVersion = "8.0")]
-    public interface IConfigureLogging
+    public partial interface IConfigureLogging
```

Comment:
This has been made partial because you're still trying to include the interfaces in the source package?